File: app/src/main/java/net/ducksmanager/persistence/models/coa/InducksCountryName.java
package net.ducksmanager.persistence.models.coa;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "inducks_countryname")
public class InducksCountryName {

    @PrimaryKey
    @NonNull
    private String countryCode;

    @ColumnInfo
    private String countryName;

    public InducksCountryName(@NonNull String countryCode, String countryName) {
        this.countryCode = countryCode;
        this.countryName = countryName;
    }

    @NonNull
    public String getCountryCode() {
        return countryCode;
    }

    public String getCountryName() {
        return countryName;
    }
}

File: app/src/main/java/net/ducksmanager/persistence/models/coa/InducksCountryName.kt
package net.ducksmanager.persistence.models.coa

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "inducks_countryname")
class InducksCountryName(
    @PrimaryKey
    val countryCode: String,

    @ColumnInfo
    val countryName: String
)